webdriver-spec.html

```diff
@@ -8814,196 +8887,68 @@
-    in the document with a <code>name</code> attribute matching
-    the value of <var>element</var>'s <code>usemap</code>
-    attribute, run these substeps:
-
-    <ol>
-     <li><p>Run the <a>element displayed</a> algorithm on <var>image element</var>
-      and set <var>any images visible</var> to <var>any images visible</var> bitwise OR its return value.
-    </ol>
-
-   <li><p>If <var>any images visible</var> is true,
-    abort these steps and return its value.
- </ol>
-
- <li><p>If <var>element</var> is an <a><code>area</code> element</a>:
-
-  <ol>
-   <li><p>For each ancestral element <var>parent</var>,
-    in <a>tree order</a>:
-
-   <ol>
-    <li><p>If <var>parent</var> is a <a><code>map</code> element</a>,
-     apply the <a>element displayed</a> algorithm on it.
-
-    <li><p>If the return value is false,
-     abort these steps and return that value.
-
-    <li><p>Otherwise apply step 7.1 on <var>parent</var>.
-   </ol>
-  </ol>
-
- <li><p>If <var>element</var> is a <a><code>text</code> node</a>,
-  return true.
-
- <li><p>If it has equal to or more than one direct descendant elements:
-
-  <ol>
-   <li><p>Let <var>visible children</var> be a boolean initially set to false.
-
-   <li><p>For each direct descendant element <var>child</var>:
-
-   <ol>
-    <li><p>Let <var>rectangle</var> be <var>child</var>’s <a>bounding rectangle</a>.
-
-    <li><p>If the value of the <code>height</code> property of <var>rectangle</var> is
-     greater than zero <a>CSS reference pixels</a>,
-     and the value of the <code>width</code> property of <var>rectangle</var>
-     is greater than zero <a>CSS reference pixels</a>:
-
-     <ol>
-      <li><p>Set <var>visible children</var> to <var>visible children</var> bitwise OR true.
-     </ol>
-   </ol>
-  </ol>
-
- <li><p>For each ancestral element <var>parent</var>, in <a>tree order</a>:
-
-  <ol>
-   <li><p>Apply the <a>element displayed</a> algorithm to <var>parent</var>.
-
-   <li><p>If the return value is false, abort these steps and return that value.
-
-   <li><p>If <var>parent</var> is a block element box and the
-    computed values of either <code>overflow-x</code>
-    or <code>overflow-y</code> is "hidden":
-
-   <ol>
-     <li><p>Let <var>parent dimensions</var> be the <a>rectangle</a>
-     that is the first element of the <a><code>DOMRect</code></a>
-     <a>sequence</a> returned by calling <a>getClientRects</a> on <var>parent</var>.
-
-    <li><p>Let <var>element dimensions</var> be the <a>rectangle</a>
-     that is the first element of the <code>DOMRect</code>
-     <a>sequence</a> returned by calling <a>getClientRects</a> on <var>element</var>.
-
-    <li><p>Let <var>parent style</var> be the computed style of <var>parent</var>.
-
-    <li><p>Return false if any the following conditions evaluate to false:
+<div class=note>
+<p>The function presented here is
+ the <a href="https://github.com/SeleniumHQ/selenium/blob/w3c-atoms/javascript/atoms/dom.js#L548">
```

Comment:
Should the link point to a commit instead of a remote branch name that could be deleted? (same as above)